tardis/visualization/tools/sdec_plot.py

```diff
@@ -826,6 +900,11 @@ def generate_plot_mpl(
         cmapname : str, optional
             Name of matplotlib colormap to be used for showing elements.
             Default value is "jet"
+        nelements: int
+            Number of elements to include in plot. Determined by the
+            contribution to total luminosity absorbed and emitted.
```

Comment:
```suggestion
            highest contribution to total luminosity absorbed and emitted.
```
I just want to make this clear we're picking the top contributing elements - so feel free to rephrase it whichever way is more suitable.